src/t2iapi/metric/service.proto

```diff
@@ -35,10 +36,12 @@ service MetricService {
         returns (BasicResponse);
 
     /*
-    Change the value for the metric with the provided handle and confirm the value using the device ui.
-     */
-    rpc SetUserConfirmableValue (BasicHandleRequest)
-        returns (BasicResponse);
+    For the metric with the given handle do the following at once:
+     - set the @ActivationState to the given activation value
+     - change the MetricValue/@Value and confirm the value using the device ui
```

Comment:
what about @Samples?
i am also missing the namespace e.g. "pm:"

also "set" instead of "change"

Reply:
done